TbsCore/Core/WebBrowserInfo.cs

```diff
@@ -31,7 +28,7 @@ public class WebBrowserInfo : IDisposable
         public async Task InitSelenium(Account acc, bool newAccess = true)
         {
             this.acc = acc;
-            Access access = newAccess ? acc.Access.GetNewAccess() : acc.Access.GetCurrentAccess();
+            Access.Access access = newAccess ? acc.Access.GetNewAccess() : acc.Access.GetCurrentAccess();

```

Comment:
why is this changed?

Reply:
maybe because my VS said Access is not a class it is a namespace ?